ruma-identifiers-validation/src/crypto_algorithms.rs

```diff
@@ -80,4 +131,17 @@ mod tests {
     fn parse_server_key_algorithm() {
         assert_eq!("ed25519".parse(), Ok(ServerKeyAlgorithm::Ed25519));
     }
+
+    #[test]
+    fn serialize_and_deserialize_from_display_form() {
```

Comment:
Now that it has been moved, the name of this test doesn't make so much sense anymore.

```suggestion
    fn event_encryption_algorithm_serde() {
```